meta-dts-distro/recipes-dts/dts/dts/dts-functions.sh

```diff
@@ -320,14 +320,24 @@ board_config() {
               HAVE_HEADS_FW="true"
               HEADS_REL_VER_DES="0.9.0"
               HEADS_LINK_DES="${FW_STORE_URL_DES}/MS-7D25/v${DASHARO_REL_VER_DES}/${DASHARO_REL_NAME}_v${DASHARO_REL_VER_DES}_ddr4_heads.rom"
-              HEADS_SWITCH_FLASHROM_OPT_OVERRIDE="" # we have to do full flashing
+              if [ ! -f $SE_credential_file ]; then
```

Comment:
Shouldn't it be DES_IS_LOGGED?

Reply:
@miczyg1 you are right, fixed https://github.com/Dasharo/meta-dts/pull/104/commits/ec3c044f3da2b54dde55d89380cc00c5c48f34f6